### PR TITLE
Load *query.config files from a directory, or failing that, the classpath

### PR DESCRIPTION
**Pull request by eddturner**

### CODE REVIEW

File: common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java [hunk 1]
```diff
@@ -18,8 +21,28 @@
 @Slf4j
 public class SolrQueryConfigFileReader {
     private static final String COMMENT_PREFIX = "#";
+    private static final String SPRING_CONFIG_LOCATION = "spring.config.location";
     private final SolrQueryConfig.SolrQueryConfigBuilder builder;
```
Comment (ahmadshadab) [inline common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java:25]: Where and how are we setting spring.config.location?

**Review by ahmadshadab — COMMENTED** [review]
Just one small clarification see my other comment

File: common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java (hunk 1 above)
Comment (eddturner) [reply common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java:25]: in the Dockerfile, i.e., the container uses `spring.config.location` to read app properties from config/meta/

Comment (ahmadshadab) [reply common-rest/src/main/java/org/uniprot/api/common/repository/search/SolrQueryConfigFileReader.java:25]: yes, all good. please merge

**Pull request merged**